vllm/model_executor/layers/pooler.py

```diff
@@ -32,7 +34,7 @@ class PoolingConfig:
         type enum value corresponding to the given string.
     """
 
-    def __init__(self, pooling_type: str, normalize: bool):
+    def __init__(self, pooling_type: str, normalize: Optional[bool] = None):
```

Comment:
Is this just to satisfy `mypy`? I think this object should not be constructed with `normalize == None`.